tests/check/tasklatencycheck_test.php

```diff
@@ -0,0 +1,216 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace tool_heartbeat;
+
+use core\check\result;
+
+/**
+ * Test class for tool_heartbeat\lib
+ *
+ * @package   tool_heartbeat
+ * @author    Jason den Dulk <jasondendulk@catalyst-au.net>
```

Comment:
I believe this is an incorrect attribution, as the tests would have likely changed from where you copied the test template from.